Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM ubuntu:14.04
+MAINTAINER Maxime Baudette "baudette@kth.se"
+# Based on Dockerfile from OM Webbook by Arunkumar Palanisamy "arunkumar.palanisamy@liu.se"
+
+RUN apt-get update
+# Install wget
+RUN apt-get install -y wget
+
+# Add OpenModelica stable build
+RUN for deb in deb deb-src; do echo "$deb http://build.openmodelica.org/apt `lsb_release -cs` stable"; done | sudo tee /etc/apt/sources.list.d/openmodelica.list
+RUN wget -q http://build.openmodelica.org/apt/openmodelica.asc -O- | sudo apt-key add -
+
+# Update index (again)
+RUN apt-get update
+
+# Install OpenModelica
+RUN apt-get install -y openmodelica
+
+RUN apt-get install -y python-pip python-dev build-essential
+RUN apt-get install -y omniidl \
+                       omniidl-python \
+                       omniorb \
+                       omniorb-idl \
+	                     git \
+					             python-numpy \
+                       python-omniorb
```

Comment:
You might wanna switch to Python3 and the new ZeroMQ interface for OMPython. Works more reliable. See also my setup: https://github.com/dietmarw/docker-notebook/blob/master/Dockerfile

Reply:
I do use ZMQ!

Reply:
Then you do not need omniorb

Reply:
the omniorb interface only works for Python 2 whilst the new zeroMQ works for Python 2 and 3. So no reason to install it.

Reply:
Yep, those lines appear as already removed. (Hence your comment showing as outdated)

Reply:
I see. Yes I should have looked at the changed files rather then the commit. But then again the commit messages were not very helpful here ;-)

Reply:
Haha. 
Yes unfortunately I did not manage to dedicate some time to setup everything to locally build docker containers or even test travis script. Also this was my first time dealing with Docker...
And yeah, OMpython and me --> not great friends
Hence, I just pushed and awaited results from dockerhub / travis to see the result. 
Sorry for the mess, this PR is ridiculous I know...